docs/pages/get-started/expo-go.mdx

```diff
@@ -49,8 +51,64 @@ The manifest is similar to an **index.html** on the web. It is served from the r
 
 Expo Go uses **SDK versions** that map to a major release of the `expo` and the Expo Go app. You can see the supported SDK versions in the Expo Go app by navigating to the settings page.
 
+When running a project that was created for an unsupported SDK version in Expo Go, you'll see the following error:
+```
+"Project is incompatible with this version of Expo Go"
+```
+To fix this, upgrading your project to a [supported SDK version](/versions/latest/#each-expo-sdk-version-depends-on-a-react-native-version) is highly recommended. If you want to learn how to do it, see [Upgrade the project to a new SDK Version](#upgrade-the-project-to-a-new-sdk-version).
+
 We release a new SDK version approximately every quarter. Find out which [versions of React Native map to which versions of the Expo SDK](/versions/latest/#each-expo-sdk-version-depends-on-a-react-native-version).
 
+#### Upgrade the project to a new SDK Version
+
+> **Info** We recommend upgrading SDK versions incrementally (one at a time). Doing so will help you pinpoint what breaks and other related issues that may occur during the upgrade process.
+
+To upgrade your project, you have to follow these steps:
+<Step label="1">
+  If you are using EAS CLI, upgrade it to the latest version:
+  <Terminal cmd={['$ npm i -g eas-cli']} />
+</Step>
+<Step label="2">
+  Install the latest version of the Expo package. For example, `yarn add expo@^49.0.0`:
+  <Tabs>
+    <Tab label="npm">
+      <Terminal cmd={['$ npm install expo@^<sdk_version>']} />
+    </Tab>
+    <Tab label="yarn">
+      <Terminal cmd={['$ yarn add expo@^<sdk_version>']} />
+    </Tab>
+  </Tabs>
+</Step>
+<Step label="3">
+  Upgrade all dependencies to match the new SDK:
+  <Terminal cmd={['$ npx expo install --fix']} />
+</Step>
+<Step label="4">
+  Check for any possible issues in your project dependencies:
+  <Terminal cmd={['$ npx expo-doctor@latest']} />
+</Step>
+
+If you encounter any issues while upgrading, see [blog post](/workflow/upgrading-expo-sdk-walkthrough/) about the SDK version. Release blog posts contain more details about the update process for each SDK version. Also, blog posts include information on any breaking changes introduced in a particular release.
+
+#### Run an unsupported project in an older version of Expo Go
+
+If you can't upgrade your project to a newer SDK version, you can still run it using an older version of Expo Go:
+<Step label="1">
+  If installed, remove Expo Go from your device/simulator.
+</Step>
+<Step label="2">
+  Start the development server:
+  <Terminal cmd={['npx expo start']}/>
+</Step>
+<Step label="3">
+  Press <kbd>a</kbd> or <kbd>i</kbd> to run your app on Android or iOS.
```

Comment:
```suggestion
  Press <kbd>a</kbd> or <kbd>i</kbd> to run your app on Android or iOS. This will download and install the compatible Expo Go on your device/simulator and open your project.
```